src/NuGet.Core/NuGet.ProjectModel/ProjectLockFile/PackagesLockFileUtilities.cs

```diff
@@ -18,6 +18,10 @@ public static class PackagesLockFileUtilities
     {
         public static bool IsNuGetLockFileEnabled(PackageSpec project)
         {
```

Comment:
There is no unit test coverage for this public method. Maybe we should add some test coverage in a follow up issue.

Reply:
Sure, but it's probably one of many, so personally not too worried as I'm sure it's covered otherwise.